### DISCUSSION TITLE
Possible to get Consumables State over REST?

### DISCUSSION THREAD

**Discussion by heilmic** [rand256/valetudo#412] [Q&A] [answered]
Hello, 

is it possible to get the Consumable State over REST?

I only get the work time of each consumable.

That's not very useful for my case. I want to notify myself, when i have to change one or more consumable. 

Or can i calculate the changing intervalls?

Thanks

**Comment by rand256** [comment, marked as answer]
Probably you're right and there should be ready to use values returned by API.
For now it returns consumables work time in seconds, and the device assumes it expired when the following values are reached:

* Main brush - 300 hours;
* Side brush - 200 hours;
* Dust filter - 150 hours;
* Sensors - 30 hours.
